zeton/templates/index_child.html

```diff
@@ -17,12 +17,14 @@
     <div class="rectangle child">
         <h2>Witaj {{ firstname }}, obecnie masz {{ points }} punktów</h2>
         <h3>Szkolny rekord tygodnia : {{ school_weekly_highscore }}</h3>
-        {% if ban %}
-            <h2 style="text-align: center; color: red;"> Masz BANA !</h2>
-            <h4> koniec bana za {{ time_ban_stop }} minuty</h4>
+
+        {% if child.bans.6.active %}
+        <h2 style="text-align: center; color: red;"> Masz BANA !</h2>
+        <h4> Ban kończy się: {{ child.bans.6.stop }}</h4>
```

Comment:
Nie kumam `child.bans.6.stop`. Czemu interesuje nas tylko zakończenie szóstego bana a nie pozostałych banów/warnów?

Reply:
Tak wymyśliłem że 6 warn czyli ban na 24h  miał być najmocniejszą formą konsekwencji. Dlatego jest uwidoczniony w górnej części strony. Wszsytkie pozostałe warny są widoczne poprzez zmianę czcionki na czerwoną w sekcji "bany_warny"

Reply:
Piąty ban też jest na 24 godziny. Jest jeszcze jeden problem: czy zawsze będzie sześć stopni banów? Bo jeśli nie, to ten kod jest błędny.

Reply:
Moim zdaniem, dzieciak w bazie powinien mieć pole "czy ma bana" jeżeli tak: wyświetlamy info, jeżeli nie to nie wyświetlamy. Można wtedy dopisać logikę, która w myśl jakichś zasad przełącza tę flagę, ale to już jest zaawansowana sprawa. A ja do końca nie wiem jak działają/mają działać bany :)

Reply:
Faktycznie - masz rację. W momencie tworzenia obsługi konsekwencji skupiłem się na obsłudze funkcjonalności banów i warnów oraz wyświetlanie ich w sekcji "bany_warny". Informację, która jest wyświetlana na górze strony tylko dostosowałem do tego aby działa poprawnie. Zmieniłem na 6 ban, bo poprzednia wersja generowała błąd - zmieniła się struktura banów. Myślę że jest to dobry pomysł na kolejny issues: Wyświetlenie najwyższego stopniem aktywnego warna oraz czas jego końca. Aczkolwiek chiałbym to przegadać z naszym wspraciem psychologicznym. Z tego względu, że powiniśmy akcentować tylko dobre strony. 
Na tą chwilę przwiduję 6 stopni banów/warnów (inna liczba za bardzo skomplikuje - tworzenie systemu żetonowego dla konkretnego dziecka). Natomiast w dalszej fazie projektu przewiduję możliwość wyłączenia modułu warnów i banów. W momencie gdy dziecko nie poradzi sobie z 6 stopniowym poziomem konsekwencji to nie warto włączać w ogóle takiego moduł do aplikacji.

Reply:
Ok. Czuję, że bany będą większą robotą :) W każdym razie, na tego PRa to nie wpływa, zatem ja daję okejkę.